android/eir/src/main/assets/form_configurations.json

```diff
@@ -7,6 +7,6 @@
   {
     "form": "record-vaccine",
     "title": "Record Vaccine",
-    "identifier": "800"
+    "identifier": "2358"
```

Comment:
This value is currently dynamically injected no?

Reply:
Yes, but the origin of the value is this file

Reply:
I see, so it can work with blank here?

Reply:
No, this is the source of the record-vaccine questionnaire identifier. If this is blank, then the app will always return null for the form